pkg/kinflate/commands/diff.go

```diff
@@ -81,23 +84,38 @@ func (o *diffOptions) RunDiff(out, errOut io.Writer, fs fs.FileSystem) error {
 		Stderr: errOut,
 	}
 
-	inflateOp := inflateOptions{manifestPath: o.manifestPath, mode: tree.ModeNormal}
```

Comment:
This concept of `ModeNormal` and `ModeNoop` should disappear altogether

Reply:
I agree with you and don't worry about this part. `ModeNormal` and `ModeNoop` are defined in package tree. This PR has removed dependency on package tree from `diff` and `inflate`. We only have three other files in package commands that depend on package tree. Once we refactor `add resource`, `set name-prefix` and `add configmap` commands, we can safely remove package tree.